### PR TITLE
[docs][develop] - match new template for "how to link permissions" and "set action permission"

### PR DESCRIPTION
**Pull request by halsaphi**
## Change Description

Update to docs:
How to link permission - match new template and working examples
cleos command reference - cleos set action permission - match new template and extended working examples

## Change Type
**Select *ONE*:**
- [ x] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Review by iamveritas — APPROVED** [review]

**Review by lparisc — APPROVED** [review]
Just a few suggestions.

File: docs/02_cleos/02_how-to-guides/how-to-link-permission.md [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
+## Before you Begin
+Make sure you meet the following requirements: 
 
-* Install the currently supported version of `cleos`
+* Install the currently supported version of `cleos.`
+[[info | Note]]
+| `Cleos` is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the `cleos` and `keosd` comand line tools. 
+* You have access to a blockchain and the `eosio.system` reference contract from [`eosio.contracts`](https://github.com/EOSIO/eosio.contracts) repository is deployed and used to manage system resources.
+* You have an EOSIO account and access to the account's `active` private key.
+* You have created a custom permssion.
```
[the remark this answers is not in the document]
Comment (halsaphi) [reply docs/02_cleos/02_how-to-guides/how-to-link-permission.md]: outdated ... this was fixed already

File: docs/02_cleos/02_how-to-guides/how-to-link-permission.md [hunk 2]
```diff
@@ -1,20 +1,44 @@
-## Goal
+## Overview
+This guide provides instructions to link a permission to a smart contract action.   
 
-Link a permission to an action of a contract
+The example uses `cleos` to link a custom permission _customp_ in the account __alice_ to a _hi_ action deployed to the _scontract_ account so that the _alice_ account's `active` permssion and _customp_ permission are authorized to call the _hi_ _action.
```
[the remark this answers is not in the document]
Comment (halsaphi) [reply docs/02_cleos/02_how-to-guides/how-to-link-permission.md]: outdated ... this was fixed already

File: docs/02_cleos/02_how-to-guides/how-to-link-permission.md [hunk 3]
```diff
@@ -1,20 +1,44 @@
...
 
-* Understand the following:
-  * What is an account
-  * What is permission level
-  * What is an action
+## Command Reference
+See the following reference guides for command line usage and related options:
 
-## Steps
+* [cleos set action permssion](../03_command-reference/set/set-action-permssion.md) command
```
[the remark this answers is not in the document]
Comment (halsaphi) [reply docs/02_cleos/02_how-to-guides/how-to-link-permission.md]: outdated ... this was fixed already

File: docs/02_cleos/03_command-reference/set/set-account-permission.md [hunk 4]
```diff
@@ -81,6 +81,21 @@ executed transaction: ab5752ecb017f166d56e7f4203ea02631e58f06f2e0b67103b71874f60
...
+3. Add a custom permission to the **alice** account
+```shell
+cleos set account permission alice customp EOS58wmANoBtT7RdPgMRCGDb37tcCQswfwVpj6NzC55D247tTMU9D active -p alice@active
+```
+**Where**
+`alice` = The name of the account you are adding a custom permssion to.
+`customp`= The name of the custom permission.
+`EOS58wmANoBtT7RdPgMRCGDb37tcCQswfwVpj6NzC55D247tTMU9D` = The public key of the custom permssion. 
+`active` = The parent of the custom permssion.
+`-p alice@active` = The permission used to authorize the transaction.
```
[the remark this answers is not in the document]
Comment (halsaphi) [reply docs/02_cleos/03_command-reference/set/set-account-permission.md]: fixed

**Pull request merged**